### ISSUE TITLE
Error while using the Features with SVM 

### ISSUE THREAD

**Issue by arshadcair** [facebook/C3D#184]
Hello,
In my work, I am using five classes of HMDB dataset (both for training and testing). The UCF101 trained network is fine-tuned using this small dataset. After the features are computed for training and test data, I use matlab script to read the features from file. I train a SVM classifier using training data. However, during testing all the test data is predicted as class label 1 (or 0 if I use 0-4 labels), For the same data when I directly test using the softmax classifier, I get an accuracy of more than 90%.

**Comment by dutran** [comment]
you may want to check your label distributions, both for training and testing. closing since it is not software-related issue.
